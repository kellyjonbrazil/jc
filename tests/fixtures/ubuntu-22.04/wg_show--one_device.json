[
  {
    "device": "wg0",
    "privateKey": "ZkG2dvJSUq5ohkDFNWBqToaVSyk4Pp8/bULaf6yX+3N=",
    "publicKey": "PQsbRs4w2fidQ7uhjgCypl5fc/804bHZbfl83gGvakN=",
    "listenPort": 51820,
    "fwmark": null,
    "peers": [
      {
        "publicKey": "rQFRAjRdx1aD8DnTFpkcBOY9Edjt3ZU7+P9HokZv7xe=",
        "presharedKey": null,
        "endpoint": "82.132.137.204:50024",
        "latestHandshake": 1829439251,
        "transferRx": 1536642,
        "transferSx": 492320,
        "persistentKeepalive": -1,
        "allowedIps": [
          "10.10.0.2/32"
        ]
      },
      {
        "publicKey": "A0fsnrwsv7S9hrk8yzCgOON9gKPYfpCrzqil4bPpxAE=",
        "presharedKey": null,
        "endpoint": "82.132.137.204:49732",
        "latestHandshake": 1829439125,
        "transferRx": 5641359,
        "transferSx": 4206783,
        "persistentKeepalive": -1,
        "allowedIps": [
          "10.10.0.3/32"
        ]
      },
      {
        "publicKey": "pkjVZR5VdfnSFhlWsqnkf+wv/eHGLHzOtKu8v2CvOD5=",
        "presharedKey": null,
        "endpoint": null,
        "latestHandshake": 0,
        "transferRx": 0,
        "transferSx": 0,
        "persistentKeepalive": -1,
        "allowedIps": [
          "10.10.0.4/32"
        ]
      },
      {
        "publicKey": "hz0+KJRHJyXfDNjUnZ9LRAnShFFjZNTrW2jRbXjIazx=",
        "presharedKey": null,
        "endpoint": "180.255.226.203:21678",
        "latestHandshake": 1829439243,
        "transferRx": 1555523,
        "transferSx": 544780,
        "persistentKeepalive": -1,
        "allowedIps": [
          "10.10.0.5/32"
        ]
      },
      {
        "publicKey": "kujr4Z8eutQ8ebzFf33DLc3PNKrO3J+ZVy3jkrR1cUZ=",
        "presharedKey": null,
        "endpoint": "82.132.137.204:67130",
        "latestHandshake": 0,
        "transferRx": 0,
        "transferSx": 0,
        "persistentKeepalive": -1,
        "allowedIps": [
          "10.10.0.6/32"
        ]
      },
      {
        "publicKey": "XcgkMRmxhQiIgXowYqsxUt1SlwUydXJd4Zmi8dgGeYe=",
        "presharedKey": null,
        "endpoint": "180.255.226.203:56092",
        "latestHandshake": 1829439121,
        "transferRx": 1900247,
        "transferSx": 1352639,
        "persistentKeepalive": -1,
        "allowedIps": [
          "10.10.0.8/32"
        ]
      },
      {
        "publicKey": "tB9yFqLmyvcDD1rJ1rTgeKtKK98hdUL1jefMRt4XbDh=",
        "presharedKey": null,
        "endpoint": "82.132.137.204:60184",
        "latestHandshake": 1829037182,
        "transferRx": 75047,
        "transferSx": 184725,
        "persistentKeepalive": -1,
        "allowedIps": [
          "10.10.0.7/32"
        ]
      }
    ]
  }
]
