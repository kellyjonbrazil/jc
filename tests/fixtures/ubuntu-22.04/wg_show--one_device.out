wg0	ZkG2dvJSUq5ohkDFNWBqToaVSyk4Pp8/bULaf6yX+3N=	PQsbRs4w2fidQ7uhjgCypl5fc/804bHZbfl83gGvakN=	51820	off
wg0	rQFRAjRdx1aD8DnTFpkcBOY9Edjt3ZU7+P9HokZv7xe=	(none)	82.132.137.204:50024	10.10.0.2/32	1829439251	1536642	492320	off
wg0	A0fsnrwsv7S9hrk8yzCgOON9gKPYfpCrzqil4bPpxAE=	(none)	82.132.137.204:49732	10.10.0.3/32	1829439125	5641359	4206783	off
wg0	pkjVZR5VdfnSFhlWsqnkf+wv/eHGLHzOtKu8v2CvOD5=	(none)	(none)	10.10.0.4/32	0	0	0	off
wg0	hz0+KJRHJyXfDNjUnZ9LRAnShFFjZNTrW2jRbXjIazx=	(none)	180.255.226.203:21678	10.10.0.5/32	1829439243	1555523	544780	off
wg0	kujr4Z8eutQ8ebzFf33DLc3PNKrO3J+ZVy3jkrR1cUZ=	(none)	82.132.137.204:67130	10.10.0.6/32	0	0	0	off
wg0	XcgkMRmxhQiIgXowYqsxUt1SlwUydXJd4Zmi8dgGeYe=	(none)	180.255.226.203:56092	10.10.0.8/32	1829439121	1900247	1352639	off
wg0	tB9yFqLmyvcDD1rJ1rTgeKtKK98hdUL1jefMRt4XbDh=	(none)	82.132.137.204:60184	10.10.0.7/32	1829037182	75047	184725	off

